src/custom/components/Web3Status/Web3StatusMod.tsx

```diff
@@ -88,19 +88,30 @@ const Web3StatusConnect = styled(Web3StatusGeneric)<{ faded?: boolean }>`
     `}
 `
 
-export const Web3StatusConnected = styled(Web3StatusGeneric)<{ pending?: boolean }>`
+export const Web3StatusConnected = styled(Web3StatusGeneric)<{ pending?: boolean; clickable?: boolean }>`
   background-color: ${({ pending, theme }) => (pending ? theme.primary1 : theme.bg2)};
   border: 1px solid ${({ pending, theme }) => (pending ? theme.primary1 : theme.bg3)};
   color: ${({ pending, theme }) => (pending ? theme.white : theme.text1)};
   font-weight: 500;
-  :hover,
-  :focus {
-    background-color: ${({ pending, theme }) => (pending ? darken(0.05, theme.primary1) : lighten(0.05, theme.bg2))};
 
-    :focus {
-      border: 1px solid ${({ pending, theme }) => (pending ? darken(0.1, theme.primary1) : darken(0.1, theme.bg3))};
-    }
-  }
+  ${({ clickable }) =>
+    clickable === false &&
+    css`
+      cursor: not-allowed;
+    `}
+
+  ${({ clickable, pending }) =>
+    clickable !== false &&
```

Comment:
if this prop isn't fed `clickable` explicitly (it's an optional prop with no default) it will still allow clicking, i.e `undefined !== false // true`

Reply:
Thank you for such detailed information.

I have moved to explicitly  `clickDisabled` state when disabling.